### DISCUSSION TITLE
Deploying HTML website build

### DISCUSSION THREAD

**Discussion by Brown-Thunder** [vercel/community#3507] [Help] [answered]
### Summary

Hi,

I have a simple HTML website and CSS file to accompany it however, when i upload my repository (which only has the HTML file, CSS file, vercel.json file and an images folder), it does not automatically display the index.html file. 

I have to set it so the directory listing is displayed so the visitor can click on index.html themself. How do i get the page to automatically display the index.html file?

I have added rewrites into my vercel.json file although I am not sure what else to do.

### Example

https://portfolio-pied-zeta-48.vercel.app/

### Steps to Reproduce

visit this [website](https://portfolio-pied-zeta-48.vercel.app/) and see the directory listing rather than index.html

**Comment by fabricio-fn** [comment]
you want the index.html to appear directly, correct?

you have a folder and inside it you have your html, css and images files, correct?

**Reply by leerob** [reply]
Appreciate your assistance here 😄

**Comment by leerob** [comment, marked as answer]
Could you try renaming the file from `Index.html` to `index.html`? I'm guessing the case sensitivity could be an issue.

For example, I just successfully deployed an HTML starter template to Vercel without any extra configuration: https://vercel.com/templates/react/html-starter.

Please let me know if this works!

**Comment by Sathyaraj-dev** [comment]
Hi,

Follow these steps for the static website deployment on Vercel.

https://stefankudla.com/posts/how-to-deploy-a-static-html-css-and-javascript-website-to-vercel

Please let me know if this works!
